firmware/controllers/system/efi_gpio.cpp

```diff
@@ -567,24 +567,6 @@ void OutputPin::initPin(const char *msg, brain_pin_e brainPin, const pin_output_
 
 #if EFI_GPIO_HARDWARE && EFI_PROD_CODE
 	efiSetPadMode(msg, brainPin, mode);
-	if (brain_pin_is_onchip(brainPin)) {
```

Comment:
in light of strange below @mck1117 please clarify this change

Reply:
damn, debugging test got left in 😢